core/cmd/renderer.go

```diff
@@ -98,13 +98,10 @@ func (rt RendererTable) renderVRFKeys(keys []VRFKeyPresenter) error {
 			key.Compressed,
 			key.Uncompressed,
 			key.Hash,
-			key.CreatedAt.String(),
```

Comment:
Do we no longer track a `created_at` time? I don't think `updated_at` is necessary, but it's always nice to see how old a key is.

Reply:
no not anymore, because all the keys are kept in a single encrypted json object. And it will be expensive to encrypt / decrypt meta data like timestamps.

Reply:
We could add this functionality back in by giving every key type a "state" like the eth keys have. But I'd rather roll that into a follow-up PR since this one is big enough as is.